core/type/src/main/java/uk/gov/gchq/gaffer/types/function/ToTypeValue.java

```diff
@@ -22,14 +22,32 @@
 
 /**
  * A {@code ToTypeValue} is a {@link KorypheFunction} that converts a
- * value into a {@link TypeValue}, by setting the Type to null
+ * value into a {@link TypeValue}, by setting the Type to null (or the provided string)
  * and the Value to the input value.
  */
 @Since("1.8.0")
 @Summary("Converts a value into a TypeValue")
 public class ToTypeValue extends KorypheFunction<Object, TypeValue> {
+    private String type;
+
+    public ToTypeValue(final String type) {
+        setType(type);
+    }
+
+    public ToTypeValue() {
```

Comment:
Is this redundant? If the default no-arg constructor is used, then the type String is null anyway?